arho_feature_template/gui/dialogs/import_features_form.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import annotations
+
+from importlib import resources
+from typing import TYPE_CHECKING, Generator
+
+from qgis.core import (
+    QgsFeature,
+    QgsFeatureIterator,
+    QgsFeatureRequest,
+    QgsFieldProxyModel,
+    QgsMapLayerProxyModel,
+    QgsVectorLayer,
+)
+from qgis.PyQt import uic
+from qgis.PyQt.QtWidgets import QCheckBox, QDialog, QDialogButtonBox, QProgressBar
+
+from arho_feature_template.core.models import PlanFeature, RegulationGroupLibrary
+from arho_feature_template.project.layers.code_layers import UndergroundTypeLayer, code_layers
+from arho_feature_template.project.layers.plan_layers import (
+    FEATURE_LAYER_NAME_TO_CLASS_MAP,
+    RegulationGroupAssociationLayer,
+    plan_feature_layers,
+    plan_layers,
+)
+from arho_feature_template.utils.misc_utils import iface, use_wait_cursor
+
+if TYPE_CHECKING:
+    from qgis.gui import QgsCheckableComboBox, QgsFieldComboBox, QgsFieldExpressionWidget, QgsMapLayerComboBox
+
+    from arho_feature_template.gui.components.code_combobox import CodeComboBox
+
+
+ui_path = resources.files(__package__) / "import_features_form.ui"
+FormClass, _ = uic.loadUiType(ui_path)
+
+
+class ImportFeaturesForm(QDialog, FormClass):
```

Comment:
Näihin pitää aina lisätä tuo ignore, kun mypy ei tykkää.
```suggestion
class ImportFeaturesForm(QDialog, FormClass):  # type: ignore
```